free-1.md

```diff
@@ -10,34 +13,75 @@ _What need is met by your idea? Who are you helping? What is that person's
+The users of the language would probably be lawyers, judges, lawmakers or reporters who wanted
+to dig into the context, history and use of various laws, definitions, rulings and interpretations.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Laws, rulings, etc. already need to be documented, and need to be accessed in a document form.
+A DSL is appropriate in that it can also supersede that use case, because it contains both
+the document form and metadata that is eminently useful for people browsing and researching
+the documents.  Additionally, it is helpful in that it could require a minimum of modification
+from the original documents to become syntactically valid (if not semantically useful).
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I came up with the project because I thought of the name "LegalEase", which has of course
+been invented a million times already.  It's exciting because I imagine that legal information
+comprises a hideously large amount of information, and it really needs a sensible means of
+managing that information via computer. (Not interpreting: law isn't black and white. 
+Just connections.)
+Especially if it could eventually be brought to the
+point of making it easier for laypeople to understand, or even point out loopholes and such.
+That's massively beyond the scope, but first there needs to be a standardized, fairly easy,
+not-going-to-be-buggy O(n) way to put the information into a format that the computer can
+manage.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Organizing and connecting legal documentation.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+A user would most likely interact with the language by using a GUI interface,
+because they are not going to be programmers-at-heart.
+However, the basic level would just be adding specific kinds of metadata,
+probably in a format similar to XML.
+
+The idea is that the original documents would already be valid, but more information
+can be added in a specific way over time to produce a more useful network of information.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+There would be many ways to run a program, more along the lines of queries.
+You can take a particular ruling, and then it would be able to take you through
+the chain of interpretations of specific laws that the ruling uses.
+You can take terms, and see how they are defined in this context, but then also
+how they are defined in a different context, and perhaps look for definition problems that way.
+You can look at the different interpretations this law a related ones have had.
```

Comment:
I'd like to see you flesh out this idea a bit more. More specifically what kind of queries might people want to see? How will they be answered? Will it give a user sixteen different opinions of the supreme court or the wikipedia one-liner?